sampleAsynchronousCode/index.js

```diff
@@ -0,0 +1,13 @@
+const randomApi = 'https://randomuser.me/api';
+
+const fetchDataFromApi = async () => {
+  try {
+    const response = await fetch(randomApi);
+    const data = await response.json();
+    console.log(data);
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+fetchDataFromApi();
```

Comment:
Hello @JanetEne, when you have an `async` function like `fetchDataFromApi`, you should use `await` when invoking the function